GPU/Software/SoftGpu.cpp

```diff
@@ -399,7 +399,7 @@ SoftGPU::SoftGPU(GraphicsContext *gfxCtx, Draw::DrawContext *draw)
 	fb.data = Memory::GetPointerWrite(0x44000000); // TODO: correct default address?
 	depthbuf.data = Memory::GetPointerWrite(0x44000000); // TODO: correct default address?
 
-	memset(softgpuCmdInfo, 0, sizeof(softgpuCmdInfo));
+	memset(&softgpuCmdInfo, 0, sizeof(softgpuCmdInfo));
```

Comment:
and here

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18971/commits/39af23aefa8517e17b9ade1d4c065c47d50901e7